tests/snmp/memory.py

```diff
@@ -1,3 +1,11 @@
 #!/usr/bin/python
-load = [' ' * 512000000]
+# memory.py <total_memory>
+import sys
+length = 512000000
+iterations = 50
+if len(sys.argv) > 1:
+    total_memory = sys.argv[1]
+    if total_memory <= 4 * 1024 * 1024:
+        length = 320000000
+load = [' ' * length]
```

Comment:
The length is based on the total memory. If the free memory is not enough, for example, less than 320000000, would it cause memory exhaustion by "load = [' ' * length]" and system crash?

Reply:
yes, that can happen, from my testing, I have not observed that happening.